doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2958,3 +2958,24 @@ The URL of an LDN Inbox to which the LDN Announce workflow step will send messag
 ++++++++++++++++++++++++++
 
 The list of parent dataset field names for which the LDN Announce workflow step should send messages. See :doc:`/developers/workflows` for details.
+
+:BasicGlobusToken
```

Comment:
Sorry, I should have put this in my quick review of the docs but why does this option start with `:Basic` when the other three start with `:Globus`? Can they all start with `:Globus`?

Reply:
fixed